cmake/luajit.cmake

```diff
@@ -0,0 +1,58 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+include_guard()
+
+include(FetchContent)
+
+FetchContent_Declare(luajit
+  GIT_REPOSITORY https://github.com/xiaobiaozhao/LuaJIT.git
```

Comment:
I think we could need to build a new repo under some organization rather than a personal one.

Reply:
Yes, I will checkout this personal repo to KvrocksLabs org

Reply:
@xiaobiaozhao Can you help to submit your changes to https://github.com/KvrocksLabs/LuaJIT?

Reply:
I pushed your commits, you can just switch to the new repo

Reply:
done https://github.com/apache/incubator-kvrocks/pull/697/commits/c4a66978d9e3c313a186ac577a98ee2a635ca8e5